src/native/src/cmake/FindGLFW3.cmake

```diff
@@ -0,0 +1,46 @@
+# Locate the glfw3 library
+#
+# This module defines the following variables:
+#
+# GLFW3_LIBRARY      the name of the library;
+# GLFW3_INCLUDE_DIR  where to find glfw include files.
+# GLFW3_FOUND        true if both the GLFW3_LIBRARY and GLFW3_INCLUDE_DIR have been found.
+#
+# To help locate the library and include file, you can define a 
+# variable called GLFW3_ROOT which points to the root of the glfw library 
+# installation.
+#
+
+# default search dirs
+set( _glfw3_HEADER_SEARCH_DIRS 
+  "/usr/include"
+  "/usr/local/include"
+  "C:/Program Files (x86)/glfw/include" )
+set( _glfw3_LIB_SEARCH_DIRS
+  "/usr/lib"
+  "/usr/local/lib"
+  "C:/Program Files (x86)/glfw/lib-msvc110" )
```

Comment:
I use CLion on top of win-builds. It works the best out of everything I've tried. By adding glfw as a subdirectory, like it was before, my compiler would build glfw, then Nova. No matter what compiler you generated the build files for, the compiler would be able to build glfw as needed. You've tied the Windows version to MSVC, which is not something we should do. A more optimal solution would be to not tie the Windows version to MSVC, and especially not to one specific version to MSVC.